docs/HACKING.md

```diff
@@ -53,7 +53,7 @@ make deploy-daemon
 
 Use `oc get pods -w` to watch for your new code to be deployed.
 
-# Hacking on MCO - updating the manifests
```

Comment:
What if we pull out the Hacking on the MCO as an upper title and shorten the sections titles as you have but make then under the Hacking one? `##`